api/serverV2.go

```diff
@@ -43,9 +44,10 @@ func NewServerV2(
 	bfIndexer blockindex.BloomFilterIndexer,
 	actPool actpool.ActPool,
 	registry *protocol.Registry,
+	getBlockTime evm.GetBlockTime,
```

Comment:
Can this be passed as an opts parameter without modifying a lot of code?

Reply:
It's not an option param, which must be used after shanghai upgrade